modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4568,6 +4568,13 @@ public DynamicCacheDescriptor cacheDescriptor(String name) {
         return cachesInfo.registeredCaches();
     }
 
+    /**
+     * @return Cache ID to Name Mapping
+     */
+    public Map<Integer, String> cacheIdToNameMapping() {
```

Comment:
Please don't expose internal data structures until it's required by anyone.

Reply:
I was following the precedence of the name - descriptor mapping. Removed now, thanks